codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/CodegenVisitor.kt

```diff
@@ -93,7 +93,10 @@ class CodegenVisitor(context: PluginContext, private val codegenDecorator: RustC
             "cargo fmt".runCommand(fileManifest.baseDir, timeout = 10)
         } catch (_: CommandFailed) {
             logger.warning("Generated output did not parse [${service.id}]")
+        } catch (ex: Exception) {
+            logger.warning("Other error trying to format: $ex")
```

Comment:
This looks like an unrelated change? May be suitable for a separate PR, or at least a call-out in this one?

Reply:
oh yeah good catch. This fixes the hard crash when `cargo` can't be found